src/node-red/getting-started/index.md

```diff
@@ -0,0 +1,14 @@
+---
+eleventyNavigation:
+  key: Getting Started
+meta:
+  title: Getting Started with Node-RED
+  description:  Learn the basics of Node-RED, a powerful tool for IoT integration and workflow automation.
+  keywords: node-red, node-red basics, node-red foundation, node-red editor
+---
+
+# {{ meta.title }}
+
+This section provides an overview of Node-RED, a robust visual tool designed for integrating IoT devices and automating workflows without needing extensive programming knowledge.
```

Comment:
Should we mention the FlowFuse Platform here?

Reply:
It's Node-RED Editor guide; I don't think we should mention.